extensions/EXT/EXT_texture_query_lod.txt

```diff
@@ -0,0 +1,183 @@
+Name
+
+    EXT_texture_query_lod
+
+Name Strings
+
+    GL_EXT_texture_query_lod
+
+Contact
+
+    Gert Wollny (gert wollny 'at' collabora.com)
+
+Contributors
+
+    Pat Brown, NVIDIA
+    Greg Roth, NVIDIA
+    Eric Werness, NVIDIA
+
+Notice
+
+    Copyright (c) 2019 Collabora LTD 
+    Copyright (c) 2009-2013 The Khronos Group Inc. Copyright terms at
+        http://www.khronos.org/registry/speccopyright.html
+
+Status
+
+    FIXME
+
+Version
+
+    Last Modified Date:         04/02/2019
+    Revision:                   1
+    Based on ARB_texture_query_lod version 4, modified 2013/10/04.
+
+Number
+
+    OpenGL ES extension #310
+
+Dependencies
+
+    OpenGL ES 3.0 is required.
+
+    OpenGL Shading Language 3.00 ES is required
+
+    EXT_gpu_shader4 is required.
+
+    EXT_texture_array is required.
+
+    This extension interacts trivially with EXT_texture_cube_map_array
+
+    This extension is written against the OpenGL ES 3.2 specification and
+    version 3.20 ES of the OpenGL Shading Language Specification.
+
+Overview
+
+    This extension adds a new set of fragment shader texture functions
+    (textureLOD) that return the results of automatic level-of-detail
+    computations that would be performed if a texture lookup were performed.
+
+New Procedures and Functions
+
+    None.
+
+New Tokens
+
+    None.
+
+Additions to the OpenGL ES 3.2 Specification
+
+    None.
+
+Errors
+
+    None.
+
+New State
+
+    None.
+
+New Implementation Dependent State
+
+    None.
+
+Modifications to The OpenGL Shading Language Specification, Version 3.20.5
+
+    Including the following line in a shader can be used to control the
+    language features described in this extension:
+
+      #extension GL_EXT_texture_query_lod
+
+    A new preprocessor #define is added to the OpenGL Shading Language:
+
+      #define GL_EXT_texture_query_lod 1
+
+    Change section 8.9.1 "Texture Query Functions"
+
+    Remove the first paragraph and add to the table:
+
+    Syntax:
+
+      vec2 textureQueryLOD(gsampler1D sampler, float coord)
```

Comment:
ES doesn't have 1D samplers.

Reply:
All reverences to 1D textures were removed,. thanks  for the review.